### PR TITLE
Update to release 1.0.3

### PR DESCRIPTION
**Pull request by monkeyswarm**

### CODE REVIEW

**Comment by monkeyswarm** [comment]
Hi @mehmetf !

**Comment by monkeyswarm** [comment]
Hi @mehmetf !

File: packages/audiofileplayer/CHANGELOG.md [hunk 1]
```diff
@@ -9,3 +9,8 @@
 ## 1.0.2 - 21 Nov 2019
 
   * Fix background audio on iOS, add ability to specify iOS audio category (which defaults to 'playback').
+
```
Comment (mehmetf) [inline packages/audiofileplayer/CHANGELOG.md:15]: Typically CHANGELOG has most recent on top.

Comment (monkeyswarm) [reply packages/audiofileplayer/CHANGELOG.md:15]: Amended with edit, thanks.

**Comment by monkeyswarm** [comment]
Hi @mehmetf !

**Review by mehmetf — APPROVED** [review]

**Pull request merged**